build_packages

```diff
@@ -222,9 +222,18 @@ break_dep_loop sys-apps/util-linux udev,systemd sys-apps/systemd cryptsetup
 break_dep_loop sys-apps/systemd cryptsetup
 
 info "Merging board packages now"
+setup_qemu_static "${BOARD_ROOT}"
 sudo -E "${EMERGE_CMD[@]}" "${EMERGE_FLAGS[@]}" \
     @system coreos-devel/board-packages
 
+# update bootengine.cpio and force rebuilt the initramfs
```

Comment:
Please put the full comment from the patch here. It tells why we are doing this.